### ISSUE TITLE
Mutex unlock just after lock

### ISSUE THREAD

**Issue by gdoffe** [cogip/mcu-firmware#65]
https://github.com/cogip/mcu-firmware/blob/4fd18e6a62ef437b98cdeb22bec0ec441ee295d4/sys/tracefd/tracefd.cpp#L34

Should `unlock()` be called at the end of the loop ?

**Comment by ecourtois** [comment]
No, it is here on purpose. That was the only way to suspend the thread I found at the time of coding this.
There should be a better solution.

**Comment by gdoffe** [comment]
Ok perfect, I will manage the thread scheduling like for the older ones (controller, planner, ...)

**Comment by gdoffe** [comment]
tracefd module has been removed, then this bug is solved by https://github.com/cogip/mcu-firmware/pull/70
